client/goal-view-ui/src/components/templates/ProovViewPage.tsx

```diff
@@ -49,12 +49,13 @@ const proofViewPage: FunctionComponent<ProofViewPageProps> = (props) => {
             <VSCodePanelView className={classes.View}> 
                 <GoalSection 
                     key={"goals"}
-                    goals={goals!.main} 
+                    goals={goals!.main}
                     collapseGoalHandler={(id) => collapseGoalHandler(id, ProofViewGoalsKey.main)} 
                     displaySetting={displaySetting}
                     emptyMessage={
                         goals!.shelved.length ? "There are shelved goals. Try using `Unshelve.`" :
                         goals!.givenUp.length ? "There are some goals you gave up. Go back and solve them, or use `Admitted.`" :
+                        goals!.unfocused.length ? "The subproof is complete, but there are some unfocused goals. Focus next goal with bullet -." :
```

Comment:
Is it maybe a bit confusing to specify `-` as the bullet type to use when it could be any of them

Reply:
Yes, I think we had a discussion with @SkySkimmer about the API to use here (or to suggest the right bullet as a quickfix if the wrong one is used)

Reply:
I'll change this to make the bullet suggestion non specific and I'll display the unfocused goals as well